components/nimbus/ios/Nimbus/FeatureHolder.swift

```diff
@@ -63,6 +63,16 @@ public class FeatureHolder<T> {
         cachedValue = value
     }
 
+    /// This resets the SDK and clears the cached value.
+    ///
+    /// This is especially useful at start up and for imported features.
+    public func with(sdk: @escaping () -> FeaturesInterface?) {
```

Comment:
Awesome! Since we are making changes to the `FeatureHolder` lets make a small note in the Changlogs that consumers must upgrade **both** the megazord and the fml version together

Reply:
Oh, flap. Yes you're right.